sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -518,6 +519,11 @@ public static partial class MultipartResponse
         public static Azure.Response[] Parse(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
         public static System.Threading.Tasks.Task<Azure.Response[]> ParseAsync(Azure.Response response, bool expectCrLf, System.Threading.CancellationToken cancellationToken) { throw null; }
     }
+    public partial class RedirectPolicyOptions
+    {
+        public RedirectPolicyOptions() { }
+        public bool IsClientRedirectEnabled { get { throw null; } set { } }
```

Comment:
I'm curious about the inclusion of `Is` in this name ... what is the thinking there vs. something like `EnableClientRedirect`?  `IsXx` implies a query regarding the state of an object (like type.IsPrimitive, e.g.) vs something a caller could pass an an input to change the behavior (e.g. `AllowSomethingToHappen`).  Is the intended usage here to ask if it is enabled?

Reply:
I don't 😄

Reply:
See boolean property naming guidelines: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-properties

Reply:
That is helpful context, thank you!

I'm interested to understand, with this guidance, how types in STJ ended up with names like ["IgnoreNullValues"](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions.ignorenullvalues?view=net-7.0).  Are they just uncompliant?